### DISCUSSION TITLE
emulate dev console command via mqtt ?

### DISCUSSION THREAD

**Discussion by lotohov** [Koenkk/zigbee2mqtt#14583] [General]
hello is anyway to emulate dev console command via mqtt ? i need this to reset Energy value on TuYa TS011F_plug_3 every day?

> `Reset energy
To reset "Sum of consumed energy", use the Dev console and execute: Endpoint: 1 Cluster: 0x00 Command: 0 Payload: (don't change this)

[#](https://www.zigbee2mqtt.io/devices/TS011F_plug_3.html`

**Comment by big-bigchen** [comment]
... Same here, looking for a way to reset the value once a day.
